Detectors/TRD/pid/macros/ccdbModelUpload.C

```diff
@@ -0,0 +1,81 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#if !defined(__CLING__) || defined(__ROOTCLING__)
+// ROOT header
+#include <TFile.h>
+// O2 header
+#include "CCDB/CcdbApi.h"
+#include "CCDB/BasicCCDBManager.h"
+#include "CCDB/CcdbObjectInfo.h"
+
+#include <map>
+#include <string>
+#include <vector>
+#include <chrono>
+#include <fstream>
+#include <ios>
+#include <iostream>
+
+#endif
+
+/// Upload an ONNX model to the ccdb.
+/// This reads the file as a binary file and stores it as such.
+void ccdbModelUpload(std::string inFileName, std::string ccdbPath = "TRD_test/PID/xgb")
+{
+
+  o2::ccdb::CcdbApi ccdb;
+  // ccdb.init("http://alice-ccdb.cern.ch");
+  // ccdb.init("http://localhost:8080");
+  ccdb.init("http://ccdb-test.cern.ch:8080");
+  // ccdb.init("http://o2-ccdb.internal");
+  std::map<std::string, std::string> metadata;
+  metadata["UploadedBy"] = "felix";
+  metadata["Description"] = "onnx model for trd pid";
+  metadata["default"] = "true"; // tag default objects
```

Comment:
Better set to `false`, or adjust time range below (should be 0..infinite for default objects)